charts/internal/e2e-resources/Chart.yaml

```diff
@@ -1,22 +1,26 @@
 apiVersion: v2
-version: 1.2.0-devel
+version: 1.3.0-devel
 description: This chart creates e2e resources for nri-kubernetes.
 name: e2e-resources
 
 dependencies:
   - name: kube-state-metrics
-    version: 2.13.2
-    repository: https://kubernetes.github.io/kube-state-metrics
+    version: 4.23.0
```

Comment:
would make sense to point to latest instead of a specific version?

Reply:
We could pin either the minor or the major 🤔

Reply:
Good point! As it is a testing chart I think we can pin the major.

Reply:
Changed here: [bb86b3f](https://github.com/newrelic/nri-kubernetes/pull/577/commits/bb86b3fa72ac3af74f33c17a4d5afed497e11ccb)